### DISCUSSION TITLE
DataTable cell word wrap with user-adjustable column widths

### DISCUSSION THREAD

**Discussion by antsin3d** [primefaces/community#141] [PrimeVue]
I'm trying to get a certain column's data to automatically wrap on load, but still let the users adjust the column widths. Whenever I enable resizing it seems to kill the wrapping (and makes the table very wide despite any attempts to limit that). I'm using this basic CSS: 

.wrap-text .p-datatable-td {
  vertical-align: top;
  word-wrap: break-word !important;
  white-space: normal !important;
  
  But haven't gotten the two to play nicely together. Anyone run into this? Thanks!

**Comment by tugcekucukoglu** [comment]
Actually, I cannot replicate the issue. Do these styles apply to table cells? Can you check it from the inspector? Also, a reproducer link really helps.
